.gitignore

```diff
@@ -8,6 +8,12 @@
 /.idea/workspace.xml
 /.idea/navEditor.xml
 /.idea/assetWizardSettings.xml
+/.idea/misc.xml
+/.idea/compiler.xml
+/.idea/gradle.xml
+/.idea/kotlinc.xml
+/.idea/vcs.xml
+/.idea/.name
```

Comment:
Which files do we really need from `/idea` directory? Seems that we can exclude whole directory instead of listing each file/subdirectory.

Reply:
Done